### PR TITLE
feat: change azure disk api-version

### PR DESCRIPTION
**Pull request by andyzhangx**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
feat: change azure disk api-version to `2019-11-01` and also changed default disk type from Standard to StandardSSD

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
feat: change azure disk api-version
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by andyzhangx** [comment]
/hold 
hold for test pass

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/diskclient/interface.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	// APIVersion is the API version for compute.
-	APIVersion = "2019-07-01"
+	APIVersion = "2019-11-01"
```
Comment (feiskyer) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/diskclient/interface.go]: should we upgrade to 2019-12-01 instead?

**Review by feiskyer — COMMENTED** [review]
/lgtm
/approve

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/diskclient/interface.go (hunk 1 above)
Comment (feiskyer) [reply staging/src/k8s.io/legacy-cloud-providers/azure/clients/diskclient/interface.go]: OK, 2019-12-01 is still not available yet on all regions

**Comment by andyzhangx** [comment]
/hold
hold a while for azure disk team to make sure `2019-11-01` is available on all regions

**Comment by andyzhangx** [comment]
also need to make sure sovereign cloud supports `2019-11-10` disks RP api-version

**Comment by andyzhangx** [comment]
/hold cancel
azure disk confirmed that new API version `2019-11-01` of disk RP is already deployed to all Azure regions, including sovereign clouds

**Pull request merged**